boards/arm/raytac_mdbt50q_db_33_nrf52833/doc/index.rst

```diff
@@ -0,0 +1,210 @@
+.. _raytac_mdbt50q_db_33_nrf52833:
+
+MDBT50Q-DB-33
```

Comment:
(optional) Might want to put Raytac at the front here so it appears with that in the documentation list